app/app.rb

```diff
@@ -27,6 +29,27 @@ def reset_date_filter
       session[:date_from] = nil
       session[:date_to] = nil
     end
+
+    def send_mail
```

Comment:
Does this want to be in a .env file? Not sure we should be displaying passwords publicly on github